presto-orc/src/main/java/com/facebook/presto/orc/DwrfWriterOptions.java

```diff
@@ -16,34 +16,22 @@
 import com.facebook.presto.orc.metadata.DwrfStripeCacheMode;
 import io.airlift.units.DataSize;
 
-import static com.facebook.presto.orc.metadata.DwrfStripeCacheMode.INDEX_AND_FOOTER;
 import static com.google.common.base.MoreObjects.toStringHelper;
-import static io.airlift.units.DataSize.Unit.MEGABYTE;
 import static java.util.Objects.requireNonNull;
 
 public class DwrfWriterOptions
 {
-    public static final DataSize DEFAULT_STRIPE_CACHE_MAX_SIZE = new DataSize(8, MEGABYTE);
-    public static final DwrfStripeCacheMode DEFAULT_STRIPE_CACHE_MODE = INDEX_AND_FOOTER;
-    private final boolean stripeCacheEnabled;
     private final DwrfStripeCacheMode stripeCacheMode;
```

Comment:
Do we still need these two fields (or the entire class) given we moved them into `OrcWriterOptions`?

Reply:
I still want to keep a class, so that the property can be grouped into a single class. I will rename this class to DwrfStripeCacheOptions in a different PR though.